test/e2e/common/node/pods.go

```diff
@@ -800,7 +800,7 @@ var _ = SIGDescribe("Pods", func() {
 		}
 
 		validatePodReadiness := func(expectReady bool) {
-			err := wait.Poll(time.Second, wait.ForeverTestTimeout, func() (bool, error) {
+			err := wait.Poll(time.Second, time.Minute, func() (bool, error) {
 				podReady := podClient.PodIsReady(podName)
```

Comment:
It is fine to increase the timeout itself.
One thing I am thinking is it is difficult to know what is actual status of the target pod according to the log of line line 807.
Current log shows `true/false` of the pod status only, I feel it is better to output actual status on the log for the investigation.

Reply:
I agree, worth improving. Will take care on the next upload.
Also, let me check this PR actually helps running a few test runs, because admittedly increasing the timeout is pretty much a blind shot.